npc/events/christmas_2006.txt

```diff
@@ -0,0 +1,4165 @@
+				monster "prt_fild05",117,155,""+strcharinfo(0)+"'s Frustration",1062,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",117,154,""+strcharinfo(0)+"'s Bombing",1062,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",118,158,""+strcharinfo(0)+"'s grudge",1246,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",118,157,""+strcharinfo(0)+"'s curse",1246,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",118,156,""+strcharinfo(0)+"'s anger",1246,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",118,155,""+strcharinfo(0)+"'s grief",1246,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",118,154,""+strcharinfo(0)+"'s hatred",1246,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",119,158,""+strcharinfo(0)+"'s a bitter taste of a single life",1245,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",119,157,""+strcharinfo(0)+"'s couple punisher",1245,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",119,156,""+strcharinfo(0)+"'s loneliness",1245,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",119,155,""+strcharinfo(0)+"'s sobbing",1245,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",119,154,""+strcharinfo(0)+"'s darkness",1245,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",120,158,""+strcharinfo(0)+"'s depression",1244,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",120,157,""+strcharinfo(0)+"'s estrangement",1244,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",120,156,""+strcharinfo(0)+"'s nightmare",1244,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",120,155,""+strcharinfo(0)+"'s wail",1244,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",120,154,""+strcharinfo(0)+"'s whisper",1244,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",121,158,""+strcharinfo(0)+"'s regret",1588,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",121,157,""+strcharinfo(0)+"'s shadow",1588,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",121,156,""+strcharinfo(0)+"'s couplebreaker",1588,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",121,155,""+strcharinfo(0)+"'s sadness",1588,1,"Enjoy#06xmas2::OnMobDead";
+				monster "prt_fild05",121,154,""+strcharinfo(0)+"'s symbol of brokenheart",1588,1,"Enjoy#06xmas2::OnMobDead";
+				end;
+			} else {
+				mes "[Enjoy]";
+				mes "There's no time to hesitate!!! Couples are enjoying Christmas out there!I believe that's not what you want!!";
+				next;
+				mes "[Enjoy]";
+				mes "What now! While you waste your ";
+				next;
+				mes "[Enjoy]";
+				mes "< ^0000ff5 Tree Knots^000000 >";
+				mes "< ^0000ff5 Wooden Blocks^000000 >";
+				mes "< ^0000ff1 Branch of Dead Tree^000000 >";
+				mes "Hurry! Bring it now!!";
+				disablenpc "Enjoy#06xmas2";
+				disablenpc "Happymerry#06xmas_02";
+				disablenpc "Christ#06xmas_02";
+				disablenpc "Mas#06mas_02";
+				disablenpc "Event#06xmas_02";
+				enablenpc "Switch#06christ_01";
+				close;
+			}
+		} else {
+			mes "[Enjoy]";
```

Comment:
This is how it appears in the official script on aegis. I can try to move around the if else statements so that it falls into that dialog but might perform a bit strangely since it has that text for two specific flag states.